src/pub_verif_token.ts

```diff
@@ -226,9 +226,9 @@ abstract class PubliclyVerifiableIssuer {
     ) {
         this.suite = (extensions?: Extensions) => {
             if (extensions === undefined) {
-                return BLIND_RSA.suite[this.mode]();
+                return BLIND_RSA.suite[this.mode](this.params);
```

Comment:
Looks reasonable, the `params` field exists in this constructor. It seems to be the right abstraction.

Reply:
it was there before. this is really a regression introduced by 0.6.0. i suspect I removed it when changing all the interfaces, and forgot to put it back